src/js/components/locations-configuration/ZoneAndBinLocations.jsx

```diff
@@ -171,12 +183,21 @@ class ZoneAndBinLocations extends Component {
 
   handleLocationEdit(values) {
     this.props.showSpinner();
-    apiClient.post(`/openboxes/api/locations/${values.id}`, {
+    const payload = values.zoneLocation ? {
```

Comment:
Instead of doing it that way, could you do it the same as in `handleSubmit` from `AddBinModal`?